r/src/array_to_vector.cpp

```diff
@@ -673,6 +676,17 @@ class Converter_Null : public Converter {
   }
 };
 
+bool arrays_can_fit_integer(ArrayVector arrays) {
```

Comment:
nit: not sure if it's worth the effort of following the C++ guide viz-a-viz function naming